frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx

```diff
@@ -33,6 +33,8 @@ interface InsightsTableProps {
     /** Key for the entityFilterLogic */
     filterKey: string
     canEditSeriesNameInline?: boolean
+    /** Whether actions that would result in the insight being updated in the backend should be disabled. */
+    disabled?: boolean
```

Comment:
`disabled` -> `readOnly` ??

Without this comment I read if as the component being disabled not things within it